src/shared/services/utils.test.js

```diff
@@ -182,25 +186,35 @@ describe('utils', () => {
       port: '8080',
       pathname: '/index.html',
       search: '',
-      hash: ''
+      hash: '',
+      username: '',
+      password: ''
     })
-    expect(utils.getUrlInfo('guides.neo4j.com')).toEqual({
```

Comment:
Can we keep this test (without protocol) since the browser will sometimes receive hostnames only from the server (when all protocols for the hostname are meant to be whitelisted)?